includes/amp-helper-functions.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+/**
```

Comment:
Missing function description, param description, and return description. We've also been adding `@since` tags.

Reply:
Your requests to document this caused me to go down the rabbit hole for the past _(far too many)_ hours.  Every time I would document something it would cause PhpStorm to flag something else as being _"off"_, so I ended up documented all of the sanitizers.  

Note I found a few places that would throw errors, e.g. `DOMNode $nodes` using methods that are only on `DOMElement` so I added a variable of logic to ensure errors are not thrown during the normal course of usage when `WP_DEBUG` is set to `true`. One such example is I added the following in several places:

    if ( ! $node instanceof DOMElement ) {
        return;
    }

I hope that was okay...

Reply:
The node type checks are probably a good idea, as I can see that clearly some of the situations would result in text nodes being used as elements. It looks like some of the instances you added would never result in a non-element being iterated (e.g. `$this->dom->getElementsByTagName( self::$tag );`) but it doesn't hurt.